plugins/view-resources/src/components/list/ListHeader.svelte

```diff
@@ -88,7 +88,7 @@
   const selection = listProvider.selection
 
   $: selectionIds = new Set($selection.map((it) => it._id))
-  $: selected = items.filter((it) => selectionIds.has(it._id))
+  $: selected = itemsProj.filter((it) => selectionIds.has(it._id))
```

Comment:
Why? Items could be less in case not all are loaded.

Reply:
it is by mistake here, should be as part of another issue